Examples/Movies/SearchScreen.js

```diff
@@ -32,7 +32,10 @@ var MovieScreen = require('./MovieScreen');
 var fetch = require('fetch');
 
 var API_URL = 'http://api.rottentomatoes.com/api/public/v1.0/';
-var API_KEYS = [/* '7waqfqbprs7pajbz28mqf6vz' - Limit Exceeded,*/ 'y4vwv8m33hed9ety83jmv52f'];
+var API_KEYS = [
```

Comment:
Can you add a comment saying that this is only for demo purpose and that it is rate limited. If you want to use it for a real app, you should create an account: http://developer.rottentomatoes.com/